src/main/java/be/syntra/devshop/DevshopFront/models/dtos/CartDto.java

```diff
@@ -12,11 +16,13 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @ToString
+@Component
+@Scope(value = "session", proxyMode = ScopedProxyMode.TARGET_CLASS)
 public class CartDto {
     private String user;
-    private LocalDateTime cartCreationDateTime;
-    private List<Product> products;
-    private boolean activeCart;
+    private LocalDateTime cartCreationDateTime = LocalDateTime.now();
+    private List<Product> products = new ArrayList<>();
+    private boolean activeCart = true;
```

Comment:
For the moment, we only use finalizedCart, when persisting. paidCart is logical, since in the future, we'll want to save the Cart when not paid. But once this object is created, it's always active, until one of the other two booleans take over. (-> when persisting). 
I say we remove this boolean from FE and BE (Cart, CartDto)

Reply:
removed in whole FE